src/pages/blog/detail.tsx

```diff
@@ -69,7 +69,7 @@ const BlogDetail = () => {
       return anchor
     })
     try {
-      const blogPath = require(`../../assets/blog/${params.blogId.toLowerCase()}.md`)
+      const blogPath = `https://misc-pages-ghost-relay.vercel.app/api/post/${params.blogId.toLowerCase()}.md?title=1`
```

Comment:
is the "?title=1" required?

Reply:
yes, the markdown by default doesn't include the title